ydb/docs/en/core/reference/kafka-api/auth.md

```diff
@@ -0,0 +1,32 @@
+# Kafka API Authentication
+
+## Enabling authentication
+
+Authentication is always enabled when using the [Kafka API in Yandex Cloud](https://yandex.cloud/docs/data-streams/kafkaapi/auth).
+
+When you run [a single-node local {{ ydb-short-name }} cluster](../../quickstart.md), [anonymous authentication](../../security/authentication.md#anonymous) is used by default.
+It doesn't require a username and password.
+To require authentication, set the [`enforce_user_token_requirement` configuration option](../configuration/index.md#auth).
```

Comment:
If the Kafka API is controlled by the same settings as the YDB-native API, I'd say exactly that and then give a link to https://ydb.tech/docs/en/security/authentication instead of trying to retell it here.